library/entity/multipart/src/main/java/org/quiltmc/qsl/multipart/impl/EntityPartTracker.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.multipart.impl;
+
+import it.unimi.dsi.fastutil.ints.Int2ObjectMap;
+import org.jetbrains.annotations.ApiStatus;
+
+import net.minecraft.entity.Entity;
+import net.minecraft.world.World;
+
+import org.quiltmc.qsl.base.api.util.InjectedInterface;
+
+@ApiStatus.Internal
+@InjectedInterface(World.class)
```

Comment:
I think this should not use interface injection as it's implementation.

I also think that the method should be `quilt$getEntityParts` if it stays implementation only.